Tests/headers/rdk/ds/dsUtl.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * If not stated otherwise in this file or this component's Licenses.txt file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2016 RDK Management
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+/**
+ * @file
+ *
+ * @brief Device Settings HAL utilities.
+ *
+ * This API defines some common utilities for the Device Settings HAL.
+ *
+ * @par Document
+ * Document reference.
+ *
+ * @par Open Issues (in no particular order)
+ * -# None
+ *
+ * @par Assumptions
+ * -# None
```

Comment:
Please do not copy pasta the whole DS in here. For example, no rdk Services use dsUTL_*. Please cut it down to only things that are required to build.

Reply:
Unit test cases need the actual rdk headers file, so included original header files.

Reply:
This request was not solved @rajnishdey  @anand-ky . Unit tests should address the rdk services. There’s no DS in that perspective, it’s external code. The purpose of those headers is to make build possible and make external calls mockable . dsUtl.h serves none of that. Same for dsDisplay.h and other declarations .  If you don’t use those declarations don’t add them. Whoever needs them for tests in future may need them in a different way. For example in tests device::List is std::vector, unlike DS. “actual“ DS is no objective and shouldn’t be. Can you fix the request @rajnishdey